main/src/library/pkger/pkger.flix

```diff
@@ -43,6 +43,39 @@ pub def removeDependency(self: Package, package: String): Package = ???
 pub def lockPackage(self: Package): List[LockedPackage] = ???
 
 // Create a new package. Everything but the name of the package can be populated with sensible defaults from the environment
-pub def initPackage(name: String): Package = ???
+pub def initPackage(_name: String): /* Package */ /*Result[Bool, Error] */ String & Impure = {
+    let test = Path.new("");
+    // TODO: ARGH! Can't consume interfaces in Flix?
```

Comment:
Working with the File interfaces rather than path seems easier at the moment.

Reply:
Ultimately I want to be able to somehow combine paths so we don't have to handle path encodings ourselves. I initially tried getting `Paths.get` to work, but when doing `import java.nio.file.Paths(String, Array[String])` we get a `[Ljava.lang.String` instead of a `java.lang.String[]` from the `Array[String]`. Putting `String[]` down doesn't work either.

Reply:
The Java type `String[]` is denoted as `Array[String]`.